lib/scenic/adapters/postgres.rb

```diff
@@ -155,6 +170,10 @@ def try_index_create(index)
         connection.execute("ROLLBACK TO SAVEPOINT #{index.index_name}")
         "index '#{index.index_name}' on '#{index.object_name}' is no longer valid and has been dropped."
       end
+
+      def raise_unless_materialized_views_supported
+        raise Scenic::Postgres::MaterializedViewsNotSupportedError unless connection.supports_materialized_views?
```

Comment:
Our styleguide prefers:

```ruby
unless connection.supports_materialized_views?
  raise #..
end.
```

This makes the conditional more obvious than when it's hiding out at the end of the line - particularly a really long one like we have here.